### ISSUE TITLE
Cors in serverless transform trips error 

### ISSUE THREAD

**Issue by mrpackethead** [aws-cloudformation/cfn-python-lint#1183]
*cfn-lint version: 0.24.8

*Description of issue.*
Attempting to use CORS in API resource causes error.

E0001 Error transforming template: Resource with id [myserviceAwsUserApi] is invalid. Cors works only with inline Swagger specified in 'DefinitionBody' property
template_clean.yaml:1:1

Commenting out the Cors on lines 141-143 removes the problem:

[myservice_clean.yaml.txt](https://github.com/aws-cloudformation/cfn-python-lint/files/3816455/myservice_clean.yaml.txt)

**Comment by kddejong** [comment]
@mrpackethead we use https://github.com/awslabs/serverless-application-model/blob/master/bin/sam-translate.py to translate a SAM template and then we lint the finished template.  The error we are seeing is actually an error from SAM translator.  

I think this looks correct.
https://github.com/awslabs/serverless-application-model/blob/4fa5d6a4ec60e82681daae6f0ff9d7669642c266/samtranslator/model/api/api_generator.py#L244

I know we are using some logic here to provide some parameter values to help make samtranslator work.  I need to validate that logic.  I'm guessing this template works for you and so this is a false positive find?

**Comment by mrpackethead** [comment]
Yes, this template works as expected, so yes this is a false positive.

**Comment by decewei** [comment]
It looks like the error still exists in Globals section in SAM

Here's a sinppet

```
Globals:
  Api:
    Cors:
      # enable option for POST preflight
      AllowMethods: "'POST,GET,OPTIONS,PATCH,DELETE'"
      AllowHeaders: "'Content-Type,X-Amz-Date,Authorization,X-Api-Key,X-Amz-Security-Token'"
      AllowOrigin: "'*'"
    GatewayResponses:
      DEFAULT_4xx:
        ResponseParameters:
          Headers:
            Access-Control-Allow-Origin: "'*'"
            Access-Control-Allow-Headers: "'Content-Type,X-Amz-Date,Authorization,X-Api-Key,X-Amz-Security-Token'"
            Access-Control-Allow-Methods: "'POST,GET,OPTIONS'"
      DEFAULT_5XX:
        ResponseParameters:
          Headers:
            Access-Control-Allow-Origin: "'*'"
            Access-Control-Allow-Headers: "'Content-Type,X-Amz-Date,Authorization,X-Api-Key,X-Amz-Security-Token'"
            Access-Control-Allow-Methods: "'POST,GET,OPTIONS'"
```

We are still getting
```
E0001 Error transforming template: Resource with id [PublicApiGateway] is invalid. Cors works only with inline Swagger specified in 'DefinitionBody' property.
/media/barrahdd/projects/sst/sst-bbx-backend/shared-infrastructures/templates/full-stack.yaml:1:1
```
this error
